CHANGELOG.md

```diff
@@ -3,7 +3,8 @@
 ### Major changes
 
 -   BREAKING: Change the recently added `transformations` callback in Python to match JavaScript API (https://github.com/pulumi/pulumi-kubernetes/pull/575)
--   BREAKING: Remove `getInputs` from Kubernetes resource implementations.
+-   BREAKING: Remove `getInputs` from Kubernetes resource implementations. (https://github.com/pulumi/pulumi-kubernetes/pull/580)
+-   BREAKING: Change provider namespace from override to default. (https://github.com/pulumi/pulumi-kubernetes/pull/585)
```

Comment:
Is there any sort of migration plan we can provide users? Folks running on `latest` tend to inevitably be surprised by these sorts of breaking changes.

Reply:
Yeah, users can still work around this change by creating a new provider instance referencing the same cluster. I'll add a comment about that to the release notes.